pkg/rbd/rbd_util.go

```diff
@@ -120,7 +120,7 @@ func createRBDImage(pOpts *rbdVolume, volSz int, adminID string, credentials map
 	}
 
 	image := pOpts.VolName
-	volSzGB := fmt.Sprintf("%dG", volSz)
+	volSzGB := fmt.Sprintf("%dM", volSz)
```

Comment:
Shouldn't this be in MiB? Also, the name of the variable doesn't reflect its contents.

Reply:
missed this one, will fix it